scripts/EnsemblREST/REST.py

```diff
@@ -0,0 +1,177 @@
+import sys
+import requests
+import json
+import math
+import time
+from tqdm import tqdm
+
+class REST(object):
+    '''
+    This class will be responsible to return data from Ensembl.
+    It will be initialized by the URL of the server. So in theory it is easy to
+    switch to alternative or to the GRCh37 server.
+    '''
+
+    def __init__(self, URL = 'https://rest.ensembl.org'):
```

Comment:
The `generate_solr_docs.py` script uses the filename "https://jul2018.rest.ensembl.org/" on Line 317 as the default due to issues with the most recent Ensembl file, should the same link be used here too?

Reply:
I removed the default REST URL from everywhere. The REST URL is set in the `generate_solr_docs.py` which needs to updated once we are moving to e95